src/ansi.rs

```diff
@@ -457,39 +457,39 @@ impl<'a, H, W> vte::Perform for Performer<'a, H, W>
             C1::NEL => self.handler.newline(),
             C1::HTS => self.handler.set_horizontal_tabstop(),
             C1::DECID => self.handler.identify_terminal(self.writer),
-            _ => err_println!("[unhandled] execute byte={:02x}", byte)
+            _ => debug!("[unhandled] execute byte={:02x}", byte)
         }
     }
 
     #[inline]
     fn hook(&mut self, params: &[i64], intermediates: &[u8], ignore: bool) {
-        err_println!("[unhandled hook] params={:?}, ints: {:?}, ignore: {:?}",
+        debug!("[unhandled hook] params={:?}, ints: {:?}, ignore: {:?}",
                      params, intermediates, ignore);
     }
 
     #[inline]
     fn put(&mut self, byte: u8) {
-        err_println!("[unhandled put] byte={:?}", byte);
+        debug!("[unhandled put] byte={:?}", byte);
     }
 
     #[inline]
     fn unhook(&mut self) {
-        err_println!("[unhandled unhook]");
+        debug!("[unhandled unhook]");
     }
 
     #[inline]
     fn osc_dispatch(&mut self, params: &[&[u8]]) {
         macro_rules! unhandled {
             () => {{
-                err_print!("[unhandled osc_dispatch]: [");
-                for param in params {
-                    err_print!("[");
-                    for byte in *param {
-                        err_print!("{:?}, ", *byte as char);
+                let mut buf = String::new();
+                for p1 in params.iter() {
+                    buf.push_str("[");
+                    for p2 in p1.iter() {
+                        buf.push_str(&format!("{:?},", *p2 as char));
                     }
-                    err_print!("],");
+                    buf.push_str("],");
                 }
-                err_println!("]");
+                debug!("[unhandled osc_dispatch]: [{}]", &buf);
```

Comment:
On second thought, maybe we should leave this as `warn!`. This isn't a hot code path so it won't be a big deal if it cannot be compiled out.